https://rt.cpan.org/Public/Bug/Display.html?id=66235

Subject: Cdnfundlibrary isn't retrieving the newest available NAV
Date: Mon Feb 28 01:39:08 2011 nul [...] ereomega.net - Ticket created

The row of the table Cdnfundlibrary retrieves is the second to most
recent (top) row. The following current code should be changed
so that the newest available NAV is retrieved.

Index: finance-quote/lib/Finance/Quote/Cdnfundlibrary.pm
===================================================================
--- finance-quote.orig/lib/Finance/Quote/Cdnfundlibrary.pm
+++ finance-quote/lib/Finance/Quote/Cdnfundlibrary.pm
@@ -92,7 +92,7 @@ sub fundlibrary   {
           $fundquote {$mutual, "method"} = "fundlibrary";
 
           # Fund price and date
-	  $row = $rows[1];
+	  $row = $rows[0];
           $fundquote {$mutual, "price"} =  $$row[2];
           $fundquote {$mutual, "nav"} = $$row[2];
           $fundquote {$mutual, "last"} = $$row[2];
